cumulus/client/relay-chain-interface/src/lib.rs

```diff
@@ -364,3 +380,18 @@ where
 		(**self).version(relay_parent).await
 	}
 }
+
+pub async fn call_remote_runtime_function<R>(
```

Comment:
Do we need this?

Reply:
It saves the caller from having to manually do the `Decode::decode`. I can remove it if you want, but I think it's useful.